bpf/aya-bpf/src/maps/mod.rs

```diff
@@ -1,3 +1,6 @@
+pub(crate) const PIN_NONE: u32 = 0;
```

Comment:
I think this could be a `#[repr(u32)] enum`?

Reply:
done